Assets/Scripts/Player/AnimationConstants.cs

```diff
@@ -11,6 +11,7 @@ public static class AnimationConstants
     public const string AnimationDownParameter = "KeyDown";
     public const string AnimationRightParameter = "KeyRight";
     public const string AnimationRunParameter = "RUN";
+    public const string AnimationBatteryGrabbingParameter = "batteryGrabbing";

```

Comment:
I feel we should have a naming convention for these parameters. But as you can see the current ones are not standardised :D 

I think changing them all to PascalCase would be nice. But it's not essential.